javadoc/src/org/netbeans/modules/javadoc/highlighting/Highlighting.java

```diff
@@ -55,21 +57,26 @@
 public class Highlighting extends AbstractHighlightsContainer implements TokenHierarchyListener {
 
     private static final Logger LOG = Logger.getLogger(Highlighting.class.getName());
-    
+    private static final String WS = " \t\n"; // NOI18N
+    private static final String JAPANESE_PERIOD = "。"; // NOI18N
```

Comment:
If the character does not fall into ANSI/ISO-8859-1 charset, better encode it as unicode escape in the java source.

Reply:
Instead of `"。"`, use `"\u3002"` i.e.  `private static final String JAPANESE_PERIOD = "\u3002";` Is this OK?

Reply:
sure